fbpic/lpa_utils/laser/antenna.py

```diff
@@ -208,25 +222,14 @@ def update_v( self, t ):
         t: float (seconds)
             The time at which to calculate the velocities
         """
```

Comment:
The code below is not needed anymore, because it is performed directly inside the `gaussian_profile` function.